data/sitesEN.json

```diff
@@ -169,6 +169,16 @@
     "destination_platform": "mediawiki",
     "destination_icon": "pikipedia.png"
   },
+  {
+    "origin": "Path of Exile Fandom Wiki",
+    "origin_base_url": "pathofexile.fandom.com",
+    "origin_content_path": "/wiki/",
+    "destination": "Path of Exile Wiki",
+    "destination_base_url": "www.poewiki.net",
+    "destination_api_path": "/wiki/",
```

Comment:
After a recent change, `destination_api_path` should now be `destination_content_path` (the value is fine)